# Type: test
# Subtype: T1
# English Summary: Update parameter call
# Chinese Summary: 更新参数调用
# Entry Number: 6

# === Git Diff Content ===

diff --git a/h2o-algos/src/test/java/hex/deeplearning/DeepLearningGradientCheck.java b/h2o-algos/src/test/java/hex/deeplearning/DeepLearningGradientCheck.java
index cb95774b8..5f71f7cb7 100644
--- a/h2o-algos/src/test/java/hex/deeplearning/DeepLearningGradientCheck.java
+++ b/h2o-algos/src/test/java/hex/deeplearning/DeepLearningGradientCheck.java
@@ -53,9 +53,9 @@ public class DeepLearningGradientCheck extends TestUtil {
               Distribution.Family.multinomial,
       }) {
         for (DeepLearningParameters.Activation act : new DeepLearningParameters.Activation[]{
-            DeepLearningParameters.Activation.ExpRectifier,
-//                DeepLearningParameters.Activation.Tanh,
-//                DeepLearningParameters.Activation.Rectifier,
+//            DeepLearningParameters.Activation.ExpRectifier,
+                DeepLearningParameters.Activation.Tanh,
+                DeepLearningParameters.Activation.Rectifier,
 //                DeepLearningParameters.Activation.Maxout,
         }) {
           for (String response : new String[]{
